wiki/Tournaments/TWC/2020/en.md

```diff
@@ -0,0 +1,225 @@
+---
+tags:
+- TWC2020
+---
+
+# osu!taiko World Cup 2020
+
+![TWC 2020 Logo](img/TWC20202x.jpg)
+
+The **osu!taiko World Cup 2020** (***TWC 2020***) is an upcoming country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It will be the tenth installment of the osu!taiko World Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2020-02-06 & 2020-02-15 |
+| Live Drawings | 2020-02-29 (14:00 UTC) |
```

Comment:
Adding UTC+x would be clearer to most users

Reply:
Sure enough